### PR TITLE
11.0 scheme

### PR DESCRIPTION
**Pull request by jarroyomorales**
Creo el PR, que así es mas fácil ir viendo los cambios

### CODE REVIEW

File: cb_maintenance/wizards/wizard_create_maintenance_request.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
                     </group>
                     <group>
                         <field name="maintenance_team_id"/>
-                        <field name="location_id"/>
+                        <field name="location_id" options="{'child_selection_field': 'name'}"/>
```
Comment (etobella) [inline cb_maintenance/wizards/wizard_create_maintenance_request.xml]: Missing widget

File: maintenance_equipment_purchase/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Creu Blanca
```
Comment (etobella) [inline maintenance_equipment_purchase/__manifest__.py]: The module should be maintenance_request_purchase,
maintenance_equipment_purchase should be used for purchase of equipments :smile:

**Review by etobella — APPROVED** [review]

**Review by etobella — CHANGES_REQUESTED** [review]
Sorry, follow my previous comments :wink:

**Comment by jarroyomorales** [comment]
@etobella Comments attended

**Pull request closed without merging**